### DISCUSSION TITLE
Disable Force Click (touch) when using SEB for MacOS

### DISCUSSION THREAD

**Discussion by pberndt12** [SafeExamBrowser/seb-mac#242] [Q&A]
Is there a way to disable Force Click (touch) in SEB to keep students from highlighting a word and force clicking it to bring up the dictionary?

**Comment by danschlet** [comment]
As the [manual](https://safeexambrowser.org/macosx/mac_usermanual_en.html#AppearancePane) states, SEB does disable Force Click by default. Doesn't that work for you? Please indicate all information necessary [mentioned here](https://safeexambrowser.org/support_en.html) (macOS version, SEB version etc) for us to reproduce this.

**Reply by pberndt12** [reply]
Our Students are using MacOS 12.4 - 12.6.1 and SEB 3.1. The screen shot is from an unconfigured SEB, but the dictionary does work within Google Classroom and Schoology. I have included the Google Classroom configuration file we use as well.
![SEBIMG](https://user-images.githubusercontent.com/117771360/200660204-03c1131a-15af-4c73-9fee-44b18f447397.jpg)
[Schoology.zip](https://github.com/SafeExamBrowser/seb-mac/files/9964641/Schoology.zip)

**Reply by danschlet** [reply]
You're right, when Look up & data detectors is set to "Force Click with one finger" in System Preferences / Trackpad / Point & Click, then look up works in the modern WebView. If your web application works with the classic WebView, you can use the setting in SEB Preferences / Browser / Select Browser Engine Policy : Force classic as a workaround. We'll try to fix this in the next update.

**Reply by pberndt12** [reply]
Sounds good. Thank you for looking into it. I will give the workaround a go and see how that works.
